### PR TITLE
add a bool value to Node which indicates if it is allowed to rez new ent...

### PR DESCRIPTION
**Pull request by sethalves**
...ities

### CODE REVIEW

File: examples/dice.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ HIFI_PUBLIC_BUCKET = "http://s3.amazonaws.com/hifi-public/";
 
 var rollSound = SoundCache.getSound(HIFI_PUBLIC_BUCKET + "sounds/dice/diceRoll.wav");
 
+var INSUFFICIENT_PERMISSIONS_ERROR_MSG = "You do not have the necessary permissions to create new Objects."
```
Comment (birarda) [inline examples/dice.js]: shouldn't `Objects` be lowercase like the rest of the sentence?

**Comment by birarda** [comment]
Looks good other than the one comment + the packet version bump

File: libraries/networking/src/PacketHeaders.cpp [hunk 2]
```diff
@@ -64,7 +64,7 @@ PacketVersion versionForPacketType(PacketType type) {
             return 2;
         case PacketTypeDomainList:
         case PacketTypeDomainListRequest:
-            return 4;
+            return 5;
```
Comment (birarda) [inline libraries/networking/src/PacketHeaders.cpp]: not sure we need to bump the request here - although it doesn't matter too much

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4e83301c65cedbd465bdd7c26c309598c9db2a6d available at: https://gist.github.com/24fd13283a0721087466<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4562.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4562.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8bca8ad4efc5221e8a73c5ddefdf5c74c40f630b available at: https://gist.github.com/24fd13283a0721087466<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4562.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4562.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 04901e99fc3f4909836e6758fa36d2d1d2085bbe available at: https://gist.github.com/b6918afa939efe4f9d82<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4562.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4562.dmg

**Pull request merged**